src/main/java/lotto/domain/LottoManager.java

```diff
@@ -0,0 +1,33 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class LottoManager {
+    private final List<Lotto> lottos;
+
+    private static final int LOTTO_PRICE = 1000;
```

Comment:
```suggestion
    private static final int LOTTO_PRICE = 1000;
    
    private final List<Lotto> lottos;
```
상수를 먼저 배치하는 것이 자바 컨벤션입니다.